yasi.py

```diff
@@ -1169,7 +1183,7 @@ def _after_indentation(res, options=None, fpath=''):
     indented_code = res['indented_code']
     indent_result = ''.join(indented_code)
     if indented_code == res['original_code'] and opts.files:
-        message = "\nFile `%s' has already been formatted. Leaving it unchanged. . .\n"
+        message = "File '%s' has already been formatted. Leaving it unchanged. . .\n"
```

Comment:
This is more of personal preference, the output is a bit harder to read with 2 line endings. Let me know if you're not a fan